src/main/java/com/redhat/exhort/providers/JavaScriptProvider.java

```diff
@@ -62,7 +62,7 @@ public abstract class JavaScriptProvider extends Provider {
 
   public JavaScriptProvider(Path manifest, Ecosystem.Type ecosystem, String cmd) {
     super(ecosystem, manifest);
-    this.cmd = Operations.getExecutable(cmd, "-v");
+    this.cmd = Operations.getExecutable(cmd, "-v", getExecEnv());
```

Comment:
Does this only affect the JS provider?

Reply:
I'm not convinced the getExecEnv is not needed when executing each command. What do you think about adding the call inside the `getExecutable` and if the existing `getExecutable` without the envs is not used, you should remove it.

Reply:
Ok, I just saw that Maven does something similar for the runProcess. I thought the `getExecEnv` functionality could be reused for other providers or that might be duplicated.
Ignore this comment.